tests/Core/Tokenizers/Tokenizer/RecurseScopeMapSwitchTokenScopeTest.inc

```diff
@@ -32,3 +32,26 @@ switch((function () {
         return 'test';
 /* testSwitchClosureWithinConditionScopeCloser */
 }
+
```

Comment:
I wonder if the below tests wouldn't have more value if they used the alternative syntax...

Reply:
I updated both tests with a return type in the switch condition to use the alternative syntax, but I kept the tests without a return type using the normal syntax as there is no `T_COLON` in the switch condition of this test.